sql/src/main/java/io/crate/operation/collect/sources/TableFunctionCollectSource.java

```diff
@@ -34,7 +34,12 @@
 import io.crate.metadata.table.TableInfo;
 import io.crate.metadata.tablefunctions.TableFunctionImplementation;
 import io.crate.operation.InputFactory;
-import io.crate.operation.collect.*;
+import io.crate.operation.collect.CrateCollector;
```

Comment:
Plz revert these import changes since you didn't touch anything else in the file.